Core/OfficeDevPnP.Core/Framework/Provisioning/ObjectHandlers/ObjectListInstance.cs

```diff
@@ -123,17 +123,17 @@ public override TokenParser ProvisionObjects(Web web, ProvisioningTemplate templ
                             foreach (var fieldRef in listInfo.TemplateList.FieldRefs)
                             {
                                 var field = rootWeb.GetFieldById<Field>(fieldRef.Id);
-                                if (field != null)
+                                if (field == null) continue;

```

Comment:
True. I'm in favor of logging it and still letting it complete the process. Then you know something wasn't quite right but the rest of the work completes. What do you think?